examples/fluids/qfunctions/bc_freestream.h

```diff
@@ -154,16 +155,13 @@ CEED_QFUNCTION_HELPER CeedScalar Softplus_fwd(CeedScalar x, CeedScalar dx, CeedS
 // little or no benefit in the tests we've run thus far, thus we recommend
 // skipping this feature and just allowing recirculation.
 CEED_QFUNCTION_HELPER int RiemannOutflow(void *ctx, CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out, StateVariable state_var) {
-  // Inputs
```

Comment:
Q - does fluids consistently not use these comments? Should pick one way or the other

Reply:
It's not consistent, but I (intend) not to for every code I touch.

Reply:
Quick grepping, definitely inconsistent, but the desire is to not have them.

Reply:
Cool. Consistent is better. This will just be a point of divergence between fluids and Ratel